spec/gilded_rose_spec.js

```diff
@@ -1,7 +1,85 @@
+'use strict'
+
+
+
 describe("Gilded Rose", function() {
 
   it("should do something", function() {
-    update_quality();
-  });
+    update_quality()
+    expect(true).toEqual(true)
+  })
+
+  it('Sell in item should decrease', () =>{
+    items = [{name:'+5 Dexterity Vest', sell_in:10, quality: 20}]
+    update_quality()
+    expect(items[0].sell_in).toEqual(9) // 10 it fails
+    expect(items[0].quality).toEqual(19) // 20 it fails
+
+  })
+
+  it('Quality degrades twice as fast as sell in days is less than zero' , () =>
+  {
+    items = [{name:'+5 Dexterity Vest', sell_in:0, quality: 20}]
+    update_quality()
+    //update_quality()
+    expect(items[0].quality).toEqual(18) // Update quality is called in every test from the above
+    expect(items.filter(function (item) {
+      return item.quality < 0
+    })).toEqual([])
+
+    update_quality()
+    expect(items.filter(function (item) {
```

Comment:
This is confusing to me. There is only one item in the items list right? So why the call to filter? 😕